merkletree/pad.go

```diff
@@ -90,15 +100,19 @@ func (pad *PAD) Update(policies *Policies) {
 	pad.updateInternal(policies, pad.latestSTR.Epoch+1)
 }
 
+// Set calculates the private index of the given name and
+// inserts the new binding into the tree.
```

Comment:
To be a bit more specific about what's going on here, I would clarify that `Set` creates a new index-to-key binding, and that `Set` inserts it into the Merkle tree underlying the PAD.